### PR TITLE
[RFC] Remove all references to tiny.vim, small.vim, and mbyte.vim.

### PR DESCRIPTION
**Pull request by brcolow**
Vim creates these scripts in test1 depending on what build features
are enabled so that tests that use these features are skiped if
necessary. Because Neovim only has one type of build (and the features
+eval, +windows, and +multi-byte are enabled in this build) they are
not necessary.

Not sure if this work is necessary as these are all legacy tests, but
it does cleanup the Makefile a bit and perhaps might reduce confusion
in the future. If there are any reasons not to merge, feel free to disregard
this PR.

### CODE REVIEW

File: src/nvim/testdir/runtest.vim [hunk 1]
```diff
@@ -21,7 +21,6 @@
 " It will be called after each Test_ function.
 
 " Without the +eval feature we can't run these tests, bail out.
```
Comment (philix) [inline src/nvim/testdir/runtest.vim]: You should also remove the comment.

Comment (brcolow) [inline src/nvim/testdir/runtest.vim]: :+1:

File: src/nvim/testdir/test52.in [hunk 2]
```diff
@@ -61,5 +60,5 @@ STARTTEST
 ENDTEST
 
 1 utf-8 text: Для Vim version 6.2.  Последнее изменение: 1970 Jan 01
-2 cp1251 text: ��� Vim version 6.2.  ��������� ���������: 1970 Jan 01
-3 cp866 text: ��� Vim version 6.2.  ��᫥���� ���������: 1970 Jan 01
+2 cp1251 text: ��� Vim version 6.2.  ��������� ���������: 1970 Jan 01
+3 cp866 text: ��� Vim version 6.2.  ��᫥���� ���������: 1970 Jan 01
```
Comment (justinmk) [inline src/nvim/testdir/test52.in]: file encoding changed from latin1 to utf8.

File: src/nvim/testdir/test1.in [hunk 3]
```diff
@@ -1,19 +1,8 @@
...
-If Vim was not compiled with the +windows feature, the tiny.vim script will be
-set like small.vim above.  tiny.vim is sourced by tests that require the
-+windows feature or other features that are missing in the tiny version.
-
-If Vim was not compiled with the +multi_byte feature, the mbyte.vim script will
-be set like small.vim above.  mbyte.vim is sourced by tests that require the
-+multi_byte feature.
+If the testing infrastructure is being run with too few lines and/or
+columns, create wrongtermsize file wich will be sourced by tests that
+require a certain terminal size to work properly.
```
Comment (justinmk) [inline src/nvim/testdir/test1.in]: This is mentioned on line 8 already, don't need this new paragraph.

Comment (brcolow) [inline src/nvim/testdir/test1.in]: Fixed.

File: src/nvim/testdir/test52.in (hunk 2 above)
Comment (brcolow) [inline src/nvim/testdir/test52.in]: I am not sure I know how to fix this...using `set encoding=latin1` on the file made it look much different.

File: src/nvim/testdir/test52.in [hunk 4]
```diff
@@ -61,5 +60,5 @@ STARTTEST
 ENDTEST
 
 1 utf-8 text: Для Vim version 6.2.  Последнее изменение: 1970 Jan 01
-2 cp1251 text: ��� Vim version 6.2.  ��������� ���������: 1970 Jan 01
-3 cp866 text: ��� Vim version 6.2.  ��᫥���� ���������: 1970 Jan 01
+2 cp1251 text: Äëÿ Vim version 6.2.  Ïîñëåäíåå èçìåíåíèå: 1970 Jan 01
+3 cp866 text: „«ï Vim version 6.2.  ®á«¥¤­¥¥ ¨§¬¥­¥­¨¥: 1970 Jan 01
```
Comment (justinmk) [inline src/nvim/testdir/test52.in]: what changed in this line? why?

**Comment by justinmk** [comment]
Please prefix the commit message subject, e.g. `test: ...`.

File: src/nvim/testdir/test52.in (hunk 4 above)
Comment (justinmk) [inline src/nvim/testdir/test52.in]: what changed in this line?

Comment (brcolow) [inline src/nvim/testdir/test52.in]: I don't know =(...I was trying to fix it.

**Comment by brcolow** [comment]
Commit message fixed.

File: src/nvim/testdir/test52.in (hunk 2 above)
Comment (justinmk) [inline src/nvim/testdir/test52.in]: Fixed in https://github.com/neovim/neovim/pull/4808. Will merge after build passes.

**Pull request closed without merging**